pnpm-lock.yaml

```diff
@@ -279,1227 +137,1280 @@ devDependencies:
   workbox-core: 6.5.0
   workbox-precaching: 6.5.0
   workbox-webpack-plugin: 6.5.0_webpack@4.46.0
-
+lockfileVersion: 5.2
```

Comment:
Did you use a different  pnpm version? This used to be 5.3 and in the beginning of the file

Reply:
reverted, re-run with 5.15.0